tests/test_connector.py

```diff
@@ -452,10 +453,30 @@ def test_cleanup3(self):
     def test_tcp_connector_ctor(self):
         conn = aiohttp.TCPConnector(loop=self.loop)
         self.assertTrue(conn.verify_ssl)
+        self.assertIs(conn.verify_fingerprint, None)
         self.assertFalse(conn.resolve)
         self.assertEqual(conn.family, socket.AF_INET)
         self.assertEqual(conn.resolved_hosts, {})
 
+    def test_tcp_connector_verify_fingerprint(self):
+        # sha1 fingerprint of ./sample.crt.der
+        fpgood = '7393fd3aed081d6fa9ae71391ae3c57f89e76cf9'
+        fpbad = 'badbadbadbadbadbadbadbadbadbadbadbadbad1'
+        for fp in (fpgood, fpbad):
+            conn = aiohttp.TCPConnector(loop=self.loop, verify_ssl=False,
+                                        verify_fingerprint=fp)
```

Comment:
One other thing, I found I had to pass `verify_ssl=False` to avoid a cert validation error, just want to make sure that's okay.